examples/datasets/plot_iris_dataset.py

```diff
@@ -48,8 +55,20 @@
 plt.xticks(())
 plt.yticks(())
 
-# To getter a better understanding of interaction of the dimensions
-# plot the first three PCA dimensions
+# Each point in the scatter plot refers to one of the 150 iris flowers
+# in the dataset, with the color indicating their respective type
+# (Setosa, Versicolour, and Virginica).
+# Just based on the the 2 dimensions used in this plot - sepal width
+# and sepal length - you can already see a patttern, but there's still
```

Comment:
```suggestion
# and sepal length - you can already see a pattern, but there's still
```

Reply:
argh the 't' on my keyboard is stuck 😅 I found some more and removed them, thanks!